dynawo/sources/Modeler/DataInterface/PowSyblIIDM/test/TestLineInterface.cpp

```diff
@@ -185,5 +185,17 @@ TEST(DataInterfaceTest, Line) {
   ASSERT_DOUBLE_EQ(li2.getQ1(), 0.0);
   ASSERT_DOUBLE_EQ(li2.getP2(), 0.0);
   ASSERT_DOUBLE_EQ(li2.getQ2(), 0.0);
+
+  MySecondLine.getTerminal1().setP(std::nan("not a number"));
```

Comment:
Useless, NaN is suposed to be the default value. Please set it to a real value or add it in the list below.

Reply:
I don't understand what to do. "nan" is not the same as "infinite". std/c++11 considers "inf" as a number. I add that line to others below anyway which are related to "inf". Or I don't see what default value you think of...

Reply:
if P is not set it is equals to Nan.I don't understand why you mention infinite here as it is not used anywhere.

Reply:
The default case about P is done line 147...
There must be something else. I add a test on IIDM object. getP() returns nan in powsybl. And our getP() turns it to 0.0. That is what I tested with std::isnan( std::nan()). There might be a bug here, in DYNLineInterfaceIIDM.cpp. I take a look on old code.

Reply:
This behavior is normal. 
Let me restart from my initial request: I would like a test, with getInitialConnected==false and P/Q != NaN to make sure the getters still return 0 in this case.

Reply:
done, in fact I missed all tests while disconnected. Corrected now :)